assignments/stretch-function-conversion.js

```diff
@@ -2,22 +2,31 @@
 
 // let myFunction = function () {};
 
+let myFunction= () => {};
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => param; 
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+let add = (param1, param2) => param1 + param2;
 
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
+let subtract = (param1, param2) => param1 - param2;
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+const exampleArray = [1,2,3,4];
+const triple = exampleArray.map(num => num * 3);
```

Comment:
Awesome usage of arrow function with map, especially when you know you can just write it without "return" on certain problems.